servicetalk-benchmarks/build.gradle

```diff
@@ -43,11 +43,11 @@ dependencies {
 }
 
 jmh {
-  include = ".*Benchmark"
+  includes = [".*Benchmark"]
   jmhVersion = "$jmhCoreVersion"
-  jvmArgsPrepend = "-Dio.netty.maxDirectMemory=9223372036854775807 " +
-                   "-Djmh.executor=CUSTOM " +
-                   "-Djmh.executor.class=io.servicetalk.benchmark.concurrent.AsyncContextFriendlyExecutor"
+  jvmArgsPrepend = ["-Dio.netty.maxDirectMemory=9223372036854775807 " +
```

Comment:
The plugin was reworked. It uses a new method of running JMH now, which does not cause issues with gradle cache, but the API for configuration is a bit buggy. For example, it requires an array now, but still needs manual string assembly instead of adding properties as separate array elements.